jparest/src/main/java/uk/gov/homeoffice/digital/sas/jparest/config/ApplicationConfig.java

```diff
@@ -8,10 +8,12 @@
 import uk.gov.homeoffice.digital.sas.jparest.exceptions.exceptionhandling.ApiResponseExceptionHandler;
 import uk.gov.homeoffice.digital.sas.jparest.swagger.PathItemCreator;
 import uk.gov.homeoffice.digital.sas.jparest.swagger.ResourceOpenApiCustomiser;
+import uk.gov.homeoffice.digital.sas.jparest.utils.EntityValidator;
 
 @Configuration
 @Import({
         ObjectMapperConfig.class,
+        EntityValidator.class,
```

Comment:
This is not needed. Actually Java configs should be imported, not spring components

Reply:
removed.